README.md

```diff
@@ -23,10 +23,10 @@ ROS messages and services used across many ROS packages in the project.
 | `boat_cmd`             | BoatCmd        | Controller               | CANTransceiverROSIntf                       |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
 | `data_sensors`         | GenericSensors | CANTransceiverROSIntf    | Local/Remote Transceiver                    |
-| `global_path`          | Path           | Local/Remote Transceiver | Local Pathfinding                           |
+| `mock_gps`             | GPS            | Boat Simulator           | CanSimIntf                                  |
```

Comment:
Let's make the mock topics after the common ones (ais ships, gps, wind sensors)

Reply:
Good point, I've done that.